test/e2e-website/material-docs.spec.ts

```diff
@@ -112,16 +112,23 @@ test.describe('Material docs', () => {
       await expect(firstAnchor).toHaveAttribute('href', '/material-ui/api/button/');
     });
 
-    test('should have correct API links when linking Base UI components', async ({ page }) => {
-      await page.goto('/material-ui/react-click-away-listener/');
-
-      const anchors = page.locator('div > h2#api ~ ul a');
-
-      const firstAnchor = anchors.first();
-      const textContent = await firstAnchor.textContent();
-
-      await expect(textContent).toEqual('<ClickAwayListener />');
-      await expect(firstAnchor).toHaveAttribute('href', '/base/api/click-away-listener/');
+    ['ClickAwayListener', 'NoSsr', 'Portal', 'TextareaAutosize'].forEach((component) => {
+      test(`should have correct API link when linking Base UI component ${component}`, async ({
+        page,
+      }) => {
+        await page.goto(`/material-ui/react-${kebabCase(component || '')}/`);
```

Comment:
End to end tests are slow, why not write a test case on the text output of the markdown logic?